src/content/h1/engine/renderer/readme.md

```diff
@@ -24,11 +41,14 @@ The renderer needed to be adapted for the range of user hardware for the PC port
 * The [fog][] screen layers effect for simulated volumetric fog does not render at all.
 * HUD shield meters are missing their flash effect when drained.
 * Monochrome bitmaps and p8 bump map formats are unsupported.
+* Custom Edition skips rendering some effects when they are not loaded yet, such as the engine lens flares in a10's intro or initial bullet hole decals.
 * Transparent shaders have a host of appearance and sorting problems:
   * Ripple map mipmaps for [shader_transparent_water][shader_transparent_water#known-issues] are reversed, with smaller mipmaps being used at closer distances. MCC is not affected.
   * [shader_transparent_glass][] which use bump maps use the wrong tag for cube map reflections in Custom Edition. They should use the shader's referenced cube map, but instead use the rasterizer _vector normalization_ bitmap referenced by [globals][].
   * Transparent shaders do not sort properly behind [shader_transparent_glass][]; they will shift in front and behind each other. Part indexes allow draw order to be set manually for objects through the [gbxmodel][], but there is no such method for the [scenario_structure_bsp][]; the best you can do is flag the transparent shader to _draw before water_ since glass shaders seem to render in the same stage as water.
   * [shader_transparent_plasma][] (energy shields) [does not render correctly][shader_transparent_plasma#known-issues] on some hardware, and always incorrectly in MCC.
+  * Environment decals using the double multiply blend mode, like those on the floor of the mission a10, do not render.
```

Comment:
double multiply, or just multiply? Should double check that

Reply:
Double. I think that specific a10 decal is the only environment decal that uses double.

Reply:
Okay, thank you!